examples/voice-pipeline-agent/function_calling_weather.py

```diff
@@ -67,6 +67,8 @@ async def entrypoint(ctx: JobContext):
         vad=ctx.proc.userdata["vad"],
         stt=deepgram.STT(),
         llm=openai.LLM(),
+        # To use vertex AI LLM
+        # llm=openai.LLM.with_vertexai(project_id="your-project-id", location="us-central1"),
```

Comment:
is there a specific reason vertex would be used here? feels odd adding this example when we have a good few other adapters

Reply:
I just added for example, I should probably add another file specific to vertex with google stt and tts